dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -24,7 +24,22 @@
               @change="handleChangeFrameworkEvent"
             />
             <slot name="reportingPeriodDropdown" />
+            <div
+              class="flex align-content-start align-items-center pl-3"
+              v-if="dataType != DataTypeEnum.EutaxonomyNonFinancials && dataType !== DataTypeEnum.Sme"
+            >
+              <InputSwitch
+                class="form-field vertical-middle"
+                data-test="dataPointToggleButton"
```

Comment:
Why is the data-test label "dataPointToggleButton"? It should be more fitting and a rather unique name.

Reply:
Renamed to hideEmptyDataPointToggleButton

Reply:
I love it.